wpiutil/src/main/native/include/wpi/Chrono.h

```diff
@@ -58,6 +58,114 @@ toTimePoint(std::time_t T, uint32_t nsec) {
 
 raw_ostream &operator<<(raw_ostream &OS, sys::TimePoint<> TP);
 
+/// Format provider for TimePoint<>
```

Comment:
I don't know that we use any of the LLVM stream formatting features?  We should be preferring fmtlib anyway.